content/projects/java-specific/hello-sql-and-java/_index.md

```diff
@@ -1,3 +1,54 @@
 ---
-title: SQL + Java Helloworld
+title: SQL Extended
 ---
+
+We have had many sessions about JDBC Templates, JPA and Hibernate, so I would hope that you now have a clue of what we are talking about.
+
+Remember the scripts you submitted for this https://umuzi-org.github.io/tech-department/projects/sql/ project now we are going to do this through JAVA.
+
+## Part1
+
+1. I would like to use vanilla java **(no Springboot)** connect to your database and perform the queries in part2 but only number [1-7]
+2. I would like you to use Springboot JDBC templates to connect to your database and perform the queries in part2 but only number [1-9]
+3. Lastly I would like you to use Hibernate + JPA to connect to your database and perform the queries in part2 [1-17]
```

Comment:
It would also be good to be more explicit about actually creating the tables in the database. 

1. Make the tables (it would be good to describe the necessary functionality in words and let them figure it out. Or give them some hints because most juniors suuuuuuuck at sql table design) 
2. Make sure you can query the tables using the various tools
3. do the specific queries needed

Reply:
1. Haven't they already made the tables  from the previous project?
2. By various tools do you mean (JDBC,JPS,Hibernate) or (myslq, tableplus or similar tech)